src/app/firedancer/topology.c

```diff
@@ -337,7 +337,7 @@ fd_topo_initialize( config_t * config ) {
 
   /**/                 fd_topob_link( topo, "gossip_verif", "gossip_verif", config->tiles.verify.receive_buffer_size, FD_TPU_RAW_MTU,                1UL );
   /**/                 fd_topob_link( topo, "gossip_tower", "gossip_tower", 128UL,                                    FD_TPU_MTU,                    1UL );
-  /**/                 fd_topob_link( topo, "replay_out",   "replay_out",   128UL,                                    sizeof(fd_replay_out_t),       1UL );
+  /**/                 fd_topob_link( topo, "replay_out",   "replay_out",   64UL,                                     sizeof(fd_replay_out_t) + (sizeof(fd_replay_out_vote_state_t) * FD_RUNTIME_MAX_VOTE_ACCOUNTS),       1UL );
```

Comment:
why change to 64? 128 was a placeholder because that used to be min-depth, but really should probably be sized sensibly to be a derived bound in tandem with a starting bound like max unrooted banks

also `replay_out` was meant to be a general link multiple tiles could use. given this is very specific input to tower, it should be its own `replay_tower` link